docs/websocket.md

```diff
@@ -307,3 +307,283 @@ eg:
    }
 }
 ```
+
```

Comment:
this documentation should be in the same section as the one above for `orderbook`, not separated by the `operations` documentation

Reply:
fixed